pkg/apis/config/register.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// GroupName is the group name used in this package
+const (
+	GroupName = "aws.k8s.io"
```

Comment:
Wondering if the API here should be specific to configs, so something like `aws.config.k8s.io`. I also think we want to avoid api groups ending in `k8s.io` unless it's an official kubernetes API.

Reply:
Thinking it should be `config.aws.io` @nckturner any opinions here? @nckturner @randomvariable @justinsb any opinions here?